mannyhagman/hows-the-weather
# Feature/implement api

#### What's this PR do?
- SearchTerm from SearchBar now return data from OpenWeather API
- Installs redux-promise middleware to unwrap axios promise in action

#### How should this be manually tested?
- Pull repository
- Add console.log('Action:', action) to line 4 of  `reducer_weather.js`
- (if problems remove this.state log from render function in `SearchBar.js`
- yarn or npm install 
- yarn or npm start
- inspect browser console and unwrapped promise data should be present
